config/install/tripal.tripalfield_collection.trpcultivate_experiments.yml

```diff
@@ -139,6 +139,54 @@ fields:
             default:
               region: content
 
+    -   name: exp_dataset
+        content_type: research_experiment
+        label: Datasets
+        type: chado_property_type_default
+        description: Use this field to describe specific datasets to be attached to the experiment. You should preferentially use the importers for data. This field is meant for datasets not yet supported by our system. Make sure to describe every column of tabular data.
+        cardinality: -1
+        required: false
+        storage_settings:
+            storage_plugin_id: chado_storage
+            storage_plugin_settings:
+                base_table: project
+                prop_table: projectprop
+        settings:
+            termIdSpace: NCIT
+            termAccession: C25474
```

Comment:
Lets use [this term](https://www.ebi.ac.uk/ols4/ontologies/sio/classes/http%253A%252F%252Fsemanticscience.org%252Fresource%252FSIO_000089) instead here:

dataset
SIO:000089
A dataset is a data item that is a collection of data items.